ursh/indico_ursh/templates/url_shortener.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'layout/base.html' %}
+
+{% block page_class %}fixed-width-standalone-text-page{% endblock %}
+
+{% block title -%}
+    {% trans %}URL Shortener{% endtrans %}
+{%- endblock %}
+
+{% block content -%}
+    <div class="page-content">
+        <p>
+            {% trans -%}
+                In addition to being able to generate short URLs for predefined assets, e.g. events, contributions,
```

Comment:
I would just explain this particular page here. "You can generate short urls for any Indico URL here" - something like that. And then maybe as a tip mention that there are also quicklinks in various places to get a shorturl.